README.md

```diff
@@ -9,21 +9,35 @@
 
 [简体中文](README_zh_CN.md)
 
-OpenPAI is an open source platform that provides complete AI model training and resource management capabilities, it is easy to extend and supports on-premise, cloud and hybrid environments in various scale.
+**OpenPAI [v1.0.0](./RELEASE_NOTE.md#2020-v1.0.0) has been released!**
```

Comment:
Please change `#2020-v1.0.0` to `#may-2020-version-100`.

Reference: https://github.com/microsoft/pai/blob/pai-0.18.y/RELEASE_NOTE.md#may-2020-version-100